godtools/App/Features/AppLanguage/Domain/UseCases/SearchAppLanguageInAppLanguagesListUseCase.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SearchAppLanguageInAppLanguagesListUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 10/4/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+
+class SearchAppLanguageInAppLanguagesListUseCase {
+    
+    func getSearchResults(for searchText: String, searchingIn languageItems: [AppLanguageListItemDomainModel]) -> [AppLanguageListItemDomainModel] {
```

Comment:
Ok I changed it to a publisher, but in doing so changed how the use case gets triggered.  But now I'm realizing that may not work with what you're thinking below about having inputs instead of a publisher.  Probably need to iron it all out when we talk tomorrow.

Reply:
Hey @rachaelblue yep that UseCase is a perfect example of what I'm looking for.  The last piece is really to make a clear connection by binding the ViewModel to the View and back.  2-way directional binding.  

I think this is it.  Bind the ViewModel @Published with View @Binding and back to the UseCase.

https://github.com/CruGlobal/godtools-swift/pull/1763

```swift
searchAppLanguageInAppLanguagesListUseCase
            .getSearchResultsPublisher(for: $searchText.eraseToAnyPublisher())
            .receive(on: DispatchQueue.main)
            .assign(to: &$appLanguageSearchResults)
```